shared/ui/carousel/carouselStyles.ts

```diff
@@ -0,0 +1,31 @@
+import { Children } from 'react';
+import styled from 'styled-components';
+import { css } from 'styled-components';
+import { Swiper } from 'swiper/react';
+
+const multipleChildren = css`
+  // Look at Main.Card component padding
+  margin-right: -${({ theme }) => theme.spaceMap.lg}px;
```

Comment:
better to write like this
```suggestion
  margin-right: ${({ theme }) => -theme.spaceMap.lg}px;
```